### PR TITLE
Add a missing mkdir to the postinst script

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom build it please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4325/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4325/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4325/

File: packaging/common/cfengine-hub/postinstall.sh [hunk 1]
```diff
@@ -147,6 +147,7 @@ export GIT_ASKPASS="\${SCRIPT_DIR}/git-askpass.sh"
 EOHIPPUS
 
   chown -R $MP_APACHE_USER:$MP_APACHE_USER $DCWORKDIR
+  mkdir -p "$DCWORKDIR/userworkdir"
```
Comment (vpodzime) [inline packaging/common/cfengine-hub/postinstall.sh]: I'm pretty sure this has to be done before the `chown -R` command right above.

Comment (Lex-2008) [reply packaging/common/cfengine-hub/postinstall.sh]: right, thanks!

**Comment by Lex-2008** [comment]
@cf-bottom please build it again!

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4326/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4326/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4326/

**Comment by Lex-2008** [comment]
@cf-bottom please build the _proper_ fix

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4327/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4327/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4327/

**Comment by Lex-2008** [comment]
base DT in the last build are green

**Comment by aleksandrychev** [comment]
@Lex-2008 manually tested, works ok

**Comment by vpodzime** [comment]
> base DT in the last build are green

And so are all the other DTs. :+1:

**Pull request merged**